docs/v1/P4Runtime-Spec.mdk

```diff
@@ -375,11 +375,11 @@ The following are not in scope of this specification document:
 
 Figure [#fig-reference-architecture] represents the P4Runtime Reference
 Architecture. The device or target to be controlled is at the bottom, and one or
-more controllers is shown at the top. A multi-master protocol allows more than
-one controller to participate, and a role-based arbitration scheme ensures only
-one controller has write access to each read/write entity, or the pipeline
-config itself. Any controller may perform read access to any entity or the
-pipeline config. Later sections describe this in detail. For the sake of
+more controllers is shown at the top. An active-active primary protocol allows
```

Comment:
This might not quite be right.

Reply:
I feel like "multi-master" protocol was not right to begin with. I am not sure what it's supposed to mean here. The figure shows a single primary controller. Maybe we should just replace "A multi-master protocol allows more than one controller to participate" with "More than one controller can participate".

Reply:
How about this?

"P4Runtime only allows for a single primary controller to actively write to each read/write entity. P4 entities can be divided into different groups, called roles. P4Runtime allows for a primary controller for each role, and a role-based arbitration scheme..."